openstack/identity/v3/users/requests.go

```diff
@@ -204,6 +204,45 @@ func Update(client *gophercloud.ServiceClient, userID string, opts UpdateOptsBui
 	return
 }
 
+// ChpwdOptsBuilder allows extensions to add additional parameters to
+// the ChangePassword request.
+type ChpwdOptsBuilder interface {
```

Comment:
Let's called this `ChangePasswordOptsBuilder`. It's a little long, but more clear.